fluffy/deliverables.py

```diff
@@ -13,15 +13,15 @@
 def print_deliverables(output_dir: Path,project_dir: Path, samples: list) -> None:
     """Create a deliverables YAML file"""
 
-    deliverables={"Files":[]}
+    deliverables={"files":[]}
     project_name=str(project_dir).strip("/").split("/")[-1]
     summary_path=f"{str(output_dir.absolute())}/summary.csv"
```

Comment:
Similar to the above
```suggestion
    summary_path=output_dir / "summary.csv"
```
Keep it as a Path object for as long as possible and convert to string when used if necessary